Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM python:3.6-alpine
```

Comment:
Do not use Alpine images; libc dependant Python packages might not work with musl and the image size does not matter in this case.